sentry/resource_sentry_project_test.go

```diff
@@ -60,6 +70,42 @@ func TestAccSentryProject_basic(t *testing.T) {
 	})
```

Comment:
Line 67 and 203, I believe, were used to pass tests that were failing due to this regression. 

Would you be able to delete the [ImportStateVerifyIgnore](https://www.terraform.io/plugin/sdkv2/resources/import#importstateverifyignore) statements and verify that the tests still pass?

Reply:
Whoops! My bad - as you stated in chat, the go-sentry project doesn't contain this representation so reading the project from Sentry won't elicit whether this has been set (and it probably shouldn't be able to).

Reply:
These fields don't exist in the go-sentry representation of a Project so the tests will fail if these are removed

https://github.com/jianyuan/go-sentry/blob/main/sentry/projects.go#L11-L61